ios/OpenTokReactNative/OTSessionManager.swift

```diff
@@ -399,12 +399,25 @@ class OTSessionManager: RCTEventEmitter {
         resolve(supportedCodecs)
     }
     
-    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: NSArray, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
+    @objc func forceMuteAll(_ sessionId: String, excludedStreamIds: Array<String>, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void{
         guard let session = OTRN.sharedState.sessions[sessionId] else {
             reject("event_failure", "Session ID not found", nil)
             return
         }
-        return resolve(true);
+        var excludedStreams:[OTStream] = []
```

Comment:
Doubt about this logic. Are you looping through empty array of excludedStreams.?